lms/static/js/student_account/views/AccessView.js

```diff
@@ -117,6 +118,19 @@
                     return this;
                 },
 
+                checkMultipleEnterprises: function(nextUrl) {
+                    var redirectUrl = this.multipleEnterpriseUrl + nextUrl;
+                    var edxUserNameCookie = this.getUserFromCookie().username;
+                    multipleEnterpriseInterface.check(edxUserNameCookie, redirectUrl, nextUrl);
+                },
+
+                getUserFromCookie: function() {
+                    // returns the user object from cookie
+                    var user = $.cookie('edx-user-info').replace(/\\/g, '').replace(/054/g, ',');
```

Comment:
i don't understand the necessity of these regexes... possibly worth adding a comment to explain this? i'm also surprised there isn't a general utility function to get the user info from the cookie, so maybe this function should live elsewhere as a utility?

Reply:
Added a util function separately.